app/definitions/rest/v1/auth.ts

```
@@ -0,0 +1,36 @@
+	settings: IUserSettings;
+	avatarUrl: string;
+}
+
+export interface ILoginDataResponse {
+	authToken: string;
+	userId: string;
+	me: ILoginMeResponse;
+}
+
+export type AuthEndpoints = {
+	'login': {
```

ESLint

[error] Replace `'login'` with `login`

(prettier/prettier)